Enterprise-CMCS/eAPD
# 4612 [Feature] Create component for Conditions for Enhanced Funding

Resolves #4612 

### Description
Adds the code for the form view of Conditions for Enhanced Funding

### Chromatic Link

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| the Conditions for Enhanced Funding page | the user clicks the "Review Standards and Conditions Regulation" link | the Standards and Conditions Help Drawer should open | jest |
| the Conditions for Enhanced Funding page | the Standards and Conditions Help Drawer is open and the user clicks close in the drawer | the help drawer should close | jest |
| the Conditions for Enhanced Funding page | the APD has no values | the form should show Qualification radio buttons with no values selected | jest |
| the Conditions for Enhanced Funding page | the APD has No Qualifications | the No radio button should be selected | jest |
| the Conditions for Enhanced Funding page | the APD has Yes Qualifications and a Justification | the Yes radio button should be selected and the Justification should display the Justification text | jest |
| the Conditions for Enhanced Funding page with no values | admin check is on |  the error message "Select an Enhanced Funding Qualification" should be displayed | jest |
| the Conditions for Enhanced Funding page with No Qualifications | admin check is on | no error messages should still be displayed | jest |
| the Conditions for Enhanced Funding page with Yes Qualifications | admin check is on | the error message "Provide an Enhanced Funding Justification" should be displayed | jest |

### Steps to manually verify this change

1. Open an MMIS APD
2. Open an Activity
3. Go to Conditions for Enhanced Funding
4. Verify that the page displays like the figma
5. Click the Review Standards and Conditions Regulation link
6. Verify that the Standards and Conditions Help Drawer opens
7. Verify that the text matches the ticket
8. Verify that clicking the Close button on the help drawer closes the drawer
9. Verify that selecting No works
10. Verify that selecting Yes opens the Justification text area
11. Verify that selecting No/Yes is saved by closing the APD and opening it again
12. Enter data in the Justification area
13. Verify that the Justification is saved by closing the APD and opening it again
14. Open the admin panel
15. Verify that no errors are displayed when the fields are filled out
16. Open another Activity
17. Verify that the error message is present when no Qualification has been selected
18. Verify that selecting No causes the error message to disappear
19. Verify that selecting Yes opens the Justification area and displays an error message for not Justification being present
20. Verify that entering data in the Justification area causes the error message to disappear

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when QA has


### This pull request can be merged when
